packages/network_traffic/manifest.yml

```
@@ -40,6 +40,18 @@ policy_templates:
             required: false
             show_user: false
             default: false
+          - name: with_vlans
+            type: bool
+            title: Configure BPF to account for VLAN tags
+            description: |-
+              Packetbeat automatically generates a BPF for capturing only the traffic on ports where it expects to find known protocols.
```

```suggestion
              Packetbeat automatically generates a BPF program for capturing only the traffic on ports where it expects to find known protocols.
```

or remove the `a` before `BPF`?